src/gui/PageView.cpp

```diff
@@ -545,8 +546,52 @@ auto XojPageView::onButtonReleaseEvent(const PositionInputData& pos) -> bool {
         } else {
             double zoom = xournal->getZoom();
             if (this->selection->userTapped(zoom)) {
-                SelectObject select(this);
-                select.at(pos.x / zoom, pos.y / zoom);
+                if (pos.isAltDown()) {
```

Comment:
Could we put this in `PageView::onButtonDoublePressEvent` (and only show when using the select tool)? To me, this seems more intuitive than holding alt and clicking.

Thank you for working on this!

Reply:
Never mind! See https://github.com/Technius/xournalpp/pull/1.